### PR TITLE
Explain cacheability testing in more detail

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

File: contents/debugging.adoc [hunk 1]
```diff
@@ -9,29 +9,126 @@ If you have a cache hit which you didn't expect we suggest to declare whatever c
...
+$ ./gradlew clean assemble --build-cache <3>
+
+BUILD SUCCESSFUL in 1s
+15 actionable tasks: 3 executed, 10 from cache, 2 up-to-date
+----
+<1> We want to start with an empty local cache.
+<2> Build the project once to let it populate the cache.
+<3> Build it again. This time everything cacheable should load from the just populated cache.
+
+You should see all cacheable tasks loaded from cache, while non-cacheable tasks should be executed.
```
Comment (wolfs) [inline contents/debugging.adoc]: I think this would be a good location to show a screenshot of the performance task breakdown from GE, like this:
image

**Comment by lptr** [comment]
I've added a screenshot.

File: contents/debugging.adoc [hunk 2]
```diff
@@ -9,29 +9,139 @@ If you have a cache hit which you didn't expect we suggest to declare whatever c
...
+[.screenshot]
+image::fully-cached-task-execution.png[]
+
+Again, use the below methods to <<diagnosing_cache_miss,diagnose>> and <<common-problems,fix>> cacheability issues.
+
+[[caching_relocation_test]]
+==== Testing cache relocatability
+
+Once everything loads properly while building the same checkout with the local cache enabled, it's time to see if there are any _relocation problems._
+A task is considered _relocatable_ if its output can be reused when the task is executed in a different location.
```
Comment (wolfs) [inline contents/debugging.adoc]: I think we should add a link to `<<relocatability>>` or merge the two explanations.

**Comment by lptr** [comment]
I fixed that, ready to merge?

**Pull request merged**